src/SFA.DAS.EmployerCommitmentsV2.Web/Mappers/DraftApprenticeship/DeleteDraftApprenticeshipViewModelMapper.cs

```diff
@@ -21,19 +22,31 @@ public DeleteDraftApprenticeshipViewModelMapper(ICommitmentsApiClient commitment
 
         public async Task<DeleteDraftApprenticeshipViewModel> Map(DeleteApprenticeshipRequest source)
         {
-            var cohort = await _commitmentsApiClient.GetCohort(source.CohortId);
+            async Task<(string FirstName, string LastName, bool IsLastApprenticeship)> GetDraftApprenticeship()
+            {
+                var draftApprenticeships = (await _commitmentsApiClient.GetDraftApprenticeships(source.CohortId)).DraftApprenticeships;
+                var draftApprenticeship = draftApprenticeships.FirstOrDefault(x => x.Id == source.DraftApprenticeshipId);
+                if (draftApprenticeship == null)
+                {
+                    throw new DraftApprenticeshipNotFoundException(
+                        $"Apprenticeship Id: {source.DraftApprenticeshipId} not found");
```

Comment:
Can we just change "Apprenticeship" to "DraftApprenticeship" for absolutely clarity in the logging.